### DISCUSSION TITLE
Implement all buttons in the left toolbar as (disable-able) extensions; remove co-dependencies between existing default extensions and core code

### DISCUSSION THREAD

**Discussion by skshivammahajan** [SVG-Edit/svgedit#790] [New feature request]
I have created a config.js where i override the settings for
```js
// EXTENSION CONFIG
svgEditor.setConfig({
    extensions: [
        // 'ext-overview_window.js', 'ext-markers.js', 'ext-connector.js', 'ext-eyedropper.js',
        // 'ext-shapes.js', 'ext-imagelib.js', 'ext-grid.js', 'ext-polygon.js', 'ext-star.js',
        // 'ext-panning.js', 'ext-storage.js'
    ]
    // noDefaultExtensions can only be meaningfully used in config.js or in the URL
    , noDefaultExtensions: true,
});
```
But on doing this only 4 extensions got disappeared whereas i want all the extensions to be disappeared from the left tool menu.
Can you please suggest me how to do this ?

**Comment by brettz9** [comment]
The toolbar buttons on the left-side of the page are implemented as core features, and cannot be disabled. I'll change your issue here to a request to implement them as extensions.

**Comment by brettz9** [comment]
Setting `noDefaultExtensions` also has, at a minimum, some styling issues, with icons appearing to the far right, especially noticeable when selecting an item...

**Comment by cuixiping** [comment]
Left side toolbar need a 'more...' button when window height is not enough.

**Comment by BS-eng2023** [comment]
Dear all, 
I have the problem that i can't activate the axtention 
![grafik](https://github.com/SVG-Edit/svgedit/assets/135705834/680bc15e-6529-4257-bb68-d2776d29a670), can anyone help?
